_includes/tutorials/streams-to-table/kstreams/code/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.5.0
+    hostname: zookeeper
+    container_name: zookeeper
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-enterprise-kafka:5.5.0
```

Comment:
qq: I thought we should use `cp-kafka` or `cp-confluent-server` because `cp-enterprise-kafka` will be deprecated

Reply:
yep, this is a copy/paste error, I'll change it.